posts/2022-04-04-version-1.5.0-release.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Version 1.5.0 Release
+---
+
+We are pleased to announce a new release of Monocle.
+
+This release mainly features a group and author scoped view. This new view displays relevant metrics tailored for a group or an author. Furthermore, you'll find a new
+page to search event' authors.
+
+Other notables changes:
+
+- 'TLS_NO_VERIFY' is now honored by the Gerrit crawler.
+- 'TLS_CYPERS' allows to override the OpenSSL cyphers list used by crawlers.
+
+Here are instructions to upgrade to 1.5.0:
+
+```Shell
+cd monocle
+
+# Stop the previous running version
+docker-compose stop
+docker-compose rm
+
+# Checkout last master
+git pull origin master
```

Comment:
I think this should be: `git fetch https://github.com/change-metrics/monocle && git checkout -b 1.5.0 1.5.0` 

Notice that you can pass fully qualified remote location to avoid issue with missing local name, e.g. is it `upstream`, `github` or `origin` ?

Reply:
Ok. I've tried however the "-t" must be passed to ensure new tags are fetched. I prefer to keep the original version (maybe git fetch -a is ok too to avoid to name the remote location).
Thanks for the review !